library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerView.java

```diff
@@ -1355,7 +1368,7 @@ public void onVideoSizeChanged(
         videoAspectRatio = 0;
       }
 
-      contentFrame.setAspectRatio(videoAspectRatio);
+      onDispatchVideoAspectRatio(contentFrame, videoAspectRatio);
```

Comment:
I think it's a bug that L1344 returns if contentFrame is null, because I think L1351-L1366 still need to execute in that case. We'll fix this as part of merging this change also. No need to do anything on your side.

Reply:
Makes sense!